### PR TITLE
[REG-2052] document camera state info

### PR DESCRIPTION
**Pull request by abeizer**
Added section to `data.zip` description for camera data.
Also reordered the headings here to appear in an order closer to their order in the actual json files.

image

---

Find the pull request instructions [here](https://www.notion.so/regressiongg/Pull-Request-Process-0d57a7eb582a446983edfacafa406f1e?pvs=4)

Every reviewer and the owner of the PR should consider these points in their request (feel free to copy this checklist so you can fill it out yourself in the overall PR comment)

- [ ] The code is extensible and backward compatible
- [ ] New public interfaces are extensible and open to backward compatibility in the future
- [ ] If preparing to remove a field in the future (i.e. this PR removes an argument), the argument stays but is no longer functional, and attaches a deprecation warning. A linear task is also created to track this deletion task.
- [ ] Non-critical or potentially modifiable arguments are optional
- [ ] Breaking changes and the approach to handling them have been verified with the team (in the Linear task, design doc, or PR itself)
- [ ] The code is easy to read
- [ ] Unit tests are added for expected and edge cases
- [ ] Integration tests are added for expected and edge cases
- [ ] Functions and classes are documented
- [ ] Migrations for both up and down operations are completed
- [ ] A documentation PR is created and being reviewed for anything in this PR that requires knowledge to use
- [ ] Implications on other dependent code (i.e. sample games and sample bots) is considered, mentioned, and properly handled
- [ ] Style changes and other non-blocking changes are marked as non-blocking from reviewers

### CODE REVIEW

File: docs/partials/recording-gameplay/_data-zip-partial.js [hunk 1]
```diff
@@ -11,6 +11,32 @@ Basic information about when this tick was captured within the context of the ov
...
 
+### Key Frame Flags
+
+Key Frame flags indicate that important changes occurred to the game state since the previous tick was captured.
+Flags are represented as an array of strings in the \`keyFrame\` field.
+
+| Key Frame Flag              | Description                                                                                                                                                                   |
+|:----------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| FIRST_FRAME                 | Indicates that this tick is the first tick of the recording                                                                                                                   |
+| SCENE                       | Indicates that the visible scenes are different from those in the previous tick                                                                                               |
```
Comment (RG-nAmKcAz) [inline docs/partials/recording-gameplay/_data-zip-partial.js:22]: SCENE is no longer used and can be removed

GAME_ELEMENT_RENDERER_COUNT is no longer used and can be removed

UI_ELEMENT was combined with GAME_ELEMENT , is no longer used and can be combined/removed

File: docs/partials/recording-gameplay/_data-zip-partial.js [hunk 2]
```diff
@@ -11,6 +11,32 @@ Basic information about when this tick was captured within the context of the ov
...
+| GAME_ELEMENT                | Indicates that a game element became visible to the camera or moved off-camera. This could be from spawning, de-spawning, character movement, camera rotation, etc.           |
+| GAME_ELEMENT_RENDERER_COUNT | Indicates that the number of renderers changed for a visible game element                                                                                                     |
+| UI_ELEMENT                  | Indicates that a UI element became visible to the camera or moved off-camera. This could be from spawning, de-spawning, UI component or canvas group visibility changes, etc. |
+| UI_PIXELHASH                | Indicates that the screen pixels of a UI component changed. This value is only relevant when using 3rd party UI systems like Coherent GameFace.                               |
+
```
Comment (RG-nAmKcAz) [inline docs/partials/recording-gameplay/_data-zip-partial.js:27]: END_RECORDING is a new keyframetype recently added for the 'last segment' ...  this captures the state and any buffered inputs when the user stops recording

**Review by RG-nAmKcAz — CHANGES_REQUESTED** [review]
Key Frame types have content that we no longer use/support

**Comment by abeizer** [comment]
@RG-nAmKcAz Updated table:

image

**Review by vontell — APPROVED** [review]

**Review by RG-nAmKcAz — APPROVED** [review]

**Review by addisonbgross — APPROVED** [review]
